src/Utf8Json/Formatters/StandardClassLibraryFormatters.cs

```diff
@@ -60,8 +60,19 @@ public ArraySegment<byte> Deserialize(ref JsonReader reader, IJsonFormatterResol
 
     public sealed class NullableStringFormatter : IJsonFormatter<string>, IObjectPropertyNameFormatter<string>
     {
-        public static readonly IJsonFormatter<string> Default = new NullableStringFormatter();
-
+        public static readonly IJsonFormatter<string> Default = new NullableStringFormatter(StringMutator.Original);
+        private static Func<string, string> _nameMutator;
```

Comment:
This should not be static, as it should be set per formatter. The tests will fails